src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -297,6 +301,49 @@ internal async Task ResolveAuthorityEndpointsAsync()
                 cancellationToken);
         }
 
+#if NET_CORE
+        private static string CreateJwkClaim(string keyId, string algorithm) // TODO: what about optional params like Modulus and Exponent?
+        {
+            // TODO: original SAL code that shows how to get other params
+            //var parameters = key.Rsa == null ? key.Parameters : key.Rsa.ExportParameters(false);
+            //return "{\"kty\":\"RSA\",\"n\":\"" + Base64UrlEncoder.Encode(parameters.Modulus) + "\",\"e\":\"" + Base64UrlEncoder.Encode(parameters.Exponent) + "\",\"alg\":\"" + algorithm + "\",\"kid\":\"" + key.KeyId + "\"}";
+
+
+            // return "{\"kty\":\"RSA\",\"alg\":\"" + algorithm + "\",\"kid\":\"" + keyId + "\"}";
+            return "{\"kid\":\"" + keyId + "\"}";
+
+        }
+
+        private static string CreateCnfRequest(IPoPCryptoProvider popCryptoProvider)
```

Comment:
This is the where I create a req_cnf according to https://tools.ietf.org/html/draft-ietf-oauth-pop-key-distribution-07#page-9

Reply:
Sadly eSTS replies with a bearer token :(